parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -56,4 +56,6 @@ IS_VHD={{GetVariable "isVHD"}}
 GPU_NODE={{GetVariable "gpuNode"}}
 SGX_NODE={{GetVariable "sgxNode"}}
 AUDITD_ENABLED={{GetVariable "auditdEnabled"}} 
+NEED_CONFIG_GPU_DRIVERS={{GetVariable "needConfigGPUDrivers"}} 
+ENABLE_GPU_SYSTEMD={{GetVariable "enableGPUSystemd"}} 
```

Comment:
ENABLE_GPU_DEVICE_PLUGIN_SYSTEMD

Reply:
changed to enableGPUDevicePlugin to make it consistent with the error code